src/Service.Host/client/src/components/allocations/SosAllocHeads.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Grid from '@material-ui/core/Grid';
+import {
+    SaveBackCancelButtons,
+    InputField,
+    Loading,
+    Title,
+    ErrorCard,
+    Dropdown,
+    SnackbarMessage,
+    utilities,
+    DatePicker,
+    OnOffSwitch
+} from '@linn-it/linn-form-components-library';
+import Page from '../../containers/Page';
+
+function SosAllocHeads({
+    editStatus,
+    itemError,
+    history,
+    loading,
+    snackbarVisible,
+    items,
+    setSnackbarVisible
+}) {
+    const creating = () => editStatus === 'create';
+    const viewing = () => editStatus === 'view';
+
+    return (
```

Comment:
I'm presuming this is just a placeholder for future changes?

Reply:
> I'm presuming this is just a placeholder for future changes?

yep.